RecoBTag/SoftLepton/plugins/SoftPFElectronTagInfoProducer.cc

```diff
@@ -109,7 +113,7 @@ void SoftPFElectronTagInfoProducer::produce(edm::Event& iEvent, const edm::Event
   			properties.ratioRel = recoelectron->p4().Dot(jetRef->p4()) / pjet.Mag2();
   			properties.p0Par    = boostedPPar(recoelectron->momentum(), jetRef->momentum());
 			properties.elec_mva    = recoelectron->mva_e_pi();
-			 if(std::abs(properties.sip3d>MaxSip3D)) continue;
+			 if(std::abs(properties.sip3dsig>MaxSip3Dsig)) continue;
```

Comment:
Yes, I'll add a commit to fix it. Yes, the impact should be negligible, as the cut is set by default at 200, that's why it passed unnoticed until now.